packages/next/client/image.tsx

```diff
@@ -152,17 +155,23 @@ export default function Image({
 }: ImageProps) {
   const thisEl = useRef<HTMLImageElement>(null)
 
-  // Sanity Checks:
-  // If priority and lazy are present, log an error and use priority only.
-  if (priority && lazy) {
-    if (process.env.NODE_ENV !== 'production') {
+  if (process.env.NODE_ENV !== 'production') {
+    if (!VALID_LOADING_VALUES.includes(loading)) {
+      throw new Error(
+        `Image with src "${src}" has invalid "loading" property. Provided "${loading}" should be one of ${VALID_LOADING_VALUES.join(
+          ','
+        )}.`
```

Comment:
```suggestion
        `Image with src "${src}" has invalid "loading" property. Provided "${loading}" should be one of ${VALID_LOADING_VALUES.map(val => val+'').join(
          ','
        )}.`
```
This will result in `'lazy, eager, '` since undefined isn't included in the join unless we cast it to a string

Reply:
@ijjk What do you think about `JSON.stringify(VALID_LOADING_VALUES)` instead?

Reply:
That results in `'["lazy","eager",null]'` right?

Reply:
@ijjk Indeed, I changed use array.map().join() instead 👍 

See 288478090b4ea254f36d46e29238c58ee7a69a6e